index.html

```diff
@@ -57,50 +57,50 @@ <h1 id="primary">Hey There. <br />I'm Lee</h1>
       <ul id="socialmedia" class="flexs">
         <li>
           <a class="icons" href="https://github.com/leehaney254"
-            ><img src="./assets/github.png" alt="Github link"
+            ><img src="./assets/github.png" alt="Link to my github page"
           /></a>
         </li>
         <li>
           <a
             class="icons"
             href="https://www.linkedin.com/in/leehaney-george-0a4a51178/"
-            ><img src="./assets/Linkedin.png" alt="Linkedin icon"
+            ><img src="./assets/Linkedin.png" alt="Link to my linkedin page"
           /></a>
         </li>
         <li>
           <a
             class="icons"
             href="https://app.slack.com/client/T47CT8XPG/D0487PRDM7F/rimeto_profile/U03T2ES9771"
-            ><img src="./assets/slack.png" alt="Slack icon"
+            ><img src="./assets/slack.png" alt="Link to my slack page"
           /></a>
         </li>
         <li>
           <a class="icons" href="https://twitter.com/Lee06785586"
-            ><img src="./assets/twitter.png" alt="Twitter icon"
+            ><img src="./assets/twitter.png" alt="Linkto my twitter page"
           /></a>
         </li>
         <li>
           <a class="icons" href="georgeleehaney@gmail.com"
-            ><img src="./assets/gmail.png" alt="gmail icon"
+            ><img src="./assets/gmail.png" alt="Link to my gmail page"
           /></a>
         </li>
       </ul>
     </section>
     <section id="section-bac">
       <div id="heading-second">
-        <h1 id="heads">My Recent Works</h1>
+        <h2 id="heads">My Recent Works</h2>
         <div id="recent-rule">
           <hr />
         </div>
       </div>
       <div class="card" id="card">
         <div id="img-div">
-          <img id="rec-image" src="./assets/first-card.png" alt="card image" />
-          <img id="large-image" src="./assets/large.png" alt="card image" />
+          <img id="rec-image" src="./assets/first-card.png" alt="" />
+          <img id="large-image" src="./assets/large.png" alt="" />
```

Comment:
- [ ] You added descriptive alt tags to other img tags and I would believe you forgot to add to these very tags. alt is short form of Alternative Text and it is for reasons when the image can't be accessed for any reason. The alt text would stand in for the image.